ckanext/ldap/controllers/user.py

```diff
@@ -106,7 +106,7 @@ def _login_success(self, user_name, came_from):
         '''
         session[u'ckanext-ldap-user'] = user_name
         session.save()
-        toolkit.redirect_to(u'user.logged_in', came_from=came_from)
+        toolkit.redirect_to(controller=u'user', action=u'logged_in', came_from=came_from)
```

Comment:
This reverts back to the old pylons syntax.
```suggestion
        toolkit.redirect_to(u'user.logged_in', came_from=came_from)
```